src/Fl_Check_Browser.cxx

```diff
@@ -243,7 +244,7 @@ int Fl_Check_Browser::add(char *s, int b) {
         p->prev = 0;
         p->checked = b;
         p->selected = 0;
-    p->text = strdup(s?s:"");
+    p->text = fl_strdup(s?s:"");
 
         if (b) {
                 nchecked_++;
```

Comment:
This whole section of code, and in fact other sections as well in this module have wacky indents.
I decided not to fix them in this large commit, but this whole module should probably be repaired.

Reply:
Agreed.

I checked both `src/Fl_Check_Browser.cxx` and `FL/Fl_Check_Browser.H` with `clang-format` and both can be formatted cleanly CMP compliant w/o manual intervention.

I can do this **after** we merged this PR if you agree.

Reply:
Yes, I agree.
I don't have clang-format 3.6 here (just 3.4.2) or I'd do it myself.
It's not a big file though; small enough to fix by hand, but maybe clang-format is the better choice.

Reply:
Created issue #121 to resolve. Also suggested a manually created patch which addresses some items not sure clang-format would.